utils/zerovec/src/hashmap/mod.rs

```diff
@@ -207,24 +61,19 @@ where
     K: ZeroMapKV<'a> + ?Sized + Hash + Eq,
     V: ZeroMapKV<'a> + ?Sized,
 {
-    /// Given a [`key`] return the probable index of the key or [`None`] if the key is guaranteed to be absent.
+    /// Given a `key` return the probable index of the key or [`None`] if the key is guaranteed to be absent.
     /// Another check to determine if the key matches the one at the probable index is required.
     #[inline]
     pub fn index<A>(&self, key: &A) -> Option<usize>
```

Comment:
This feels like an implementation detail, not much a client can do this this without access to `keys` and `values`.

Try to match `ZeroMap`'s API by implementing methods like `contains_key`, `iter_keys`, `iter_values`, etc. Those should be fairly simple.